final_prep/annotations.rb

```diff
@@ -2,42 +2,63 @@
 # Try to focus on using correct technical vocabulary.
 # Use the # to create a new comment
 
-# Build a Bear
-
+#Creates a method called build_a_bear that has 5 inputs
 def build_a_bear(name, age, fur, clothes, special_power)
+  #sets a variable called greeting to a string with the variable "name" interpolated into it
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  #creates an array called demographics containing name and age
   demographics = [name, age]
+  #creates a string called power_saying with special_power interpolated into it
   power_saying = "Did you know that I can #{special_power}?"
+  #creates a hash called built_bear
   built_bear = {
+    #keys basic_info to demographics array
     'basic_info' => demographics,
+    #keys 'clothes' to clothes input
     'clothes' => clothes,
```

Comment:
Nice job pointing out whether the value comes from an input, or variable declared above.